Chummer/Chummer.csproj

```diff
@@ -91,7 +91,7 @@
   </PropertyGroup>
   <PropertyGroup />
   <PropertyGroup>
-    <ApplicationManifest>Properties\app.manifest</ApplicationManifest>
```

Comment:
Uh, pretty sure we already have an app.manifest file, and it's in the location that's referred to here. Is there something with its location?

Reply:
Yes, we do. I just moved it to the main folder. I wondered if it's actually loaded, because I get get so many Windows 8 installations and the MS-Docs say, that the Environment.OSVersion returns Windows 8 if Windows 10 is not specifically supported in the app.manifest file. Do you mind moving the file?